e2e/gui/src/test/java/com/epam/pipeline/autotests/LimitMountsTest.java

```diff
@@ -0,0 +1,261 @@
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import static com.codeborne.selenide.Condition.disabled;
+import static com.codeborne.selenide.Condition.enabled;
+import static com.codeborne.selenide.Condition.exist;
+import static com.codeborne.selenide.Condition.not;
+import static com.codeborne.selenide.Condition.text;
+import static com.codeborne.selenide.Selenide.open;
+import static com.epam.pipeline.autotests.ao.LogAO.configurationParameter;
+import static com.epam.pipeline.autotests.ao.LogAO.containsMessages;
+import static com.epam.pipeline.autotests.ao.LogAO.log;
+import static com.epam.pipeline.autotests.ao.LogAO.taskWithName;
+import static com.epam.pipeline.autotests.ao.Primitive.ADVANCED_PANEL;
+import static com.epam.pipeline.autotests.ao.Primitive.CANCEL;
+import static com.epam.pipeline.autotests.ao.Primitive.CLEAR_SELECTION;
+import static com.epam.pipeline.autotests.ao.Primitive.EXEC_ENVIRONMENT;
+import static com.epam.pipeline.autotests.ao.Primitive.LIMIT_MOUNTS;
+import static com.epam.pipeline.autotests.ao.Primitive.OK;
+import static com.epam.pipeline.autotests.ao.Primitive.PARAMETERS;
+import static com.epam.pipeline.autotests.ao.Primitive.SELECT_ALL;
+import static com.epam.pipeline.autotests.ao.Primitive.SELECT_ALL_NON_SENSITIVE;
+import static com.epam.pipeline.autotests.ao.Primitive.TABLE;
+import static com.epam.pipeline.autotests.utils.Privilege.EXECUTE;
+import static com.epam.pipeline.autotests.utils.Privilege.READ;
+import static com.epam.pipeline.autotests.utils.Privilege.WRITE;
+import static com.epam.pipeline.autotests.utils.PrivilegeValue.ALLOW;
+import static java.lang.String.format;
+
+public class LimitMountsTest extends AbstractSeveralPipelineRunningTest implements Navigation, Authorization, Tools {
+
+    private String storage1 = "limitMountsStorage" + Utils.randomSuffix();
+    private String storage2 = "limitMountsStorage" + Utils.randomSuffix();
+    private String storage3 = "limitMountsStorage" + Utils.randomSuffix();
+    private String storage4 = "limitMountsStorage" + Utils.randomSuffix();
+    private final String registry = C.DEFAULT_REGISTRY;
+    private final String testTool = C.TESTING_TOOL_NAME;
+    private final String group = C.DEFAULT_GROUP;
+    private final String mountDataStoragesTask = "MountDataStorages";
+
+    @BeforeClass(alwaysRun = true)
+    public void setPreferences() {
+        library()
+                .createStorage(storage1)
+                .createStorage(storage2)
+                .createStorage(storage3)
+                .clickOnCreateStorageButton()
+                .setStoragePath(storage4)
+                .clickSensitiveStorageCheckbox()
+                .ok();
+        givePermissionsToStorage(user, storage2);
+        givePermissionsToStorage(user, storage3);
+        givePermissionsToStorage(user, storage4);
```

Comment:
Could be changed to `Stream.of(storage2, storage3, storage4).forEach(s -> givePermissionsToStorage(user, s));`

Reply:
Done